app/src/main/kotlin/pl/gov/mc/protego/ui/ViewExtensions.kt

```diff
@@ -0,0 +1,21 @@
+package pl.gov.mc.protego.ui
+
+import android.view.View
+import android.widget.EditText
+import android.widget.ScrollView
+
+fun ScrollView.scrollToBottom() {
+    val lastChild = getChildAt(childCount - 1)
+    val bottom = lastChild.bottom + paddingBottom
+    val delta = bottom - (scrollY + height)
+    smoothScrollBy(0, delta)
+}
```

Comment:
Ten kod działa w oparciu o założenia na temat tego jak wygląda UI. Jeśli layout  się zmieni, to może przestać poprawnie działać. Opakowanie w ScrollView i flaga adjustResize są ok, ale mam wątpliwości co do tego czy to jest potrzebne.

Reply:
To prawda, o ile widoki sporo by się zmieniły. IMO sam `adjustResize` nie wystarcza, żeby interfejs tych ekranów wyglądał płynnie, stąd ta zmiana która dobrze wygląda przy typowym użyciu:
- wpisanie numeru
- przejście dalej
- wpisanie kodu
- przejście dalej

Reply:
Nie znam założeń projektu ani wymagań co do UI, ale moim zdaniem z perspektywy UX lepiej byłoby dodać po prostu obsługę kliknięcia w guzik ok/done na klawiaturze (`onEditorActionListener`). Wtedy guzik nie jest tak ważny, wystarczy że pole tekstowe widać no i jest to po prostu wygodne :slightly_smiling_face: